tasks/service-runner/app/routers/router.ts

```diff
@@ -0,0 +1,19 @@
+import express, { NextFunction } from 'express';
+
+/**
+ *
+ * @returns Router configured with service routes.
+ */
+export default function router(): express.Router {
+  const result = express.Router();
+
+  result.get('/liveness', async (req, res, _next: NextFunction): Promise<void> => {
+    res.send('OK');
+  });
+
+  result.get('/readiness', async (req, res, _next: NextFunction): Promise<void> => {
+    res.send('OK');
```

Comment:
Does /readiness need to wait for the service's container to start or will things work if it hasn't started?

Reply:
/readiness is an endpoint that the k8s services checks to see if the pod is ready to handle requests from the service. Since we don't actually send requests through the service (we pull work) this endpoint doesn't really mean anything. I just kept it around because I'm not sure if the service will complain if it can't reach this endpoint from time-to-time.